app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java

```diff
@@ -50,6 +54,7 @@ public void onCreate() {
 
         DebotStrategyBuilder builder = new DebotStrategyBuilder.Builder(this)
                 .registerMenu("Clear cache", clearCache)
+                .registerMenu("Notification test ON/OFF", notificationStrategy)
```

Comment:
I'm not sure whether the debug menu tile can be changed at runtime.
It depends on Debot spec.